expense_tracker/lib/models/expense.dart

```diff
@@ -0,0 +1,96 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:uuid/uuid.dart';
+
+const uuid = Uuid();
+final formatter = DateFormat.yMd();
+
+/// Categories of expenses.
+enum Category {
+  food,
+  travel,
+  leisure,
+  work,
+}
+
+/// Icons for each expense category.
+const categoryIcons = {
+  Category.food: Icons.lunch_dining,
+  Category.travel: Icons.flight,
+  Category.leisure: Icons.movie,
+  Category.work: Icons.work,
+};
+
```

Comment:
You can actually simplify this to
`
enum Category {
    food: (icon: Icons.launch_dining),
}
`
which I figured out later. Pretty cool huh